source/simulator/assembly.cc

```diff
@@ -307,8 +325,6 @@ namespace aspect
       namespace CopyData
       {
 
-
-
```

Comment:
You often have these whitespace changes in assembly.cc. Are they by purpose or leftover from moving stuff around? In the latter case please remove the changes.